src/components/templates/PosterHall/components/PosterPreview/PosterPreview.tsx

```diff
@@ -28,7 +31,11 @@ export const PosterPreview: React.FC<PosterPreviewProps> = ({
 
   const { push: openUrlUsingRouter } = useHistory();
   const handleEnterVenue = useCallback(
-    () => enterVenue(venueId, { customOpenRelativeUrl: openUrlUsingRouter }),
+    (e) => {
+      if (e.target.closest("PosterPreview__bookmark")) return;
```

Comment:
1. `closest` takes selector as a parameter, so we should indicate that this is a class name adding `.` in the beginning (`".PosterPreview__bookmark"`)
2. would be better to make this string `"PosterPreview__bookmark"` a const as it's used 2 times in the file: to be able to update it when needed at once.

Reply:
I've addressed point 1. here f8f823a7af6a74681675e1a122557c4fae0f4277
Regarding making the string a constant, I ran into some problems there. I'll need to play around with it a bit more, but if you have any tips for dealing with the preceed `.` that differs between the two places, please let me know.

Reply:
2. you can find the example in the `hideEventSchedule` function of the `NavBar` component.
We have `navBarScheduleClassName` const there and used it this way in `closest()` :  \`.${navBarScheduleClassName} `

Reply:
Thanks! Implemented here: 7ed6c2ad1c81adfeaa09fde2708bb4c779d093f9